src/test/lib/ApiTest.cpp

```diff
@@ -2681,6 +2681,50 @@ void QuicTestGlobalParam()
         }
     }
 
+    //
+    // QUIC_PARAM_GLOBAL_PLATFORM_WORKER_POOL
```

Comment:
We need to disable this test for release build because it's not guaranteed to not break with future changes. The reason being is that future changes to expose application controlled execution context likely will result in more changes in this area.